packages/valory/connections/abci/connection.py

```diff
@@ -457,12 +458,15 @@ def __init__(self, params: TendermintParams, logger: Optional[Logger] = None):
         self._monitoring: Optional[StoppableThread] = None
         self.logger = logger or logging.getLogger()
         self.log_file = os.environ.get("LOG_FILE", DEFAULT_TENDERMINT_LOG_FILE)
+        output = subprocess.check_output(["tendermint", "version"])  # nosec
+        self.logger.info(f"Tendermint version: {output.decode(ENCODING).strip()}")
 
     def _build_init_command(self) -> List[str]:
         """Build the 'init' command."""
         cmd = [
             "tendermint",
             "init",
+            "validator",
```

Comment:
we must specify one of the following: `[validator|full|seed]`

some details: [ADR 052: Tendermint Mode](https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-052-tendermint-mode.md)